File: PodoalGlobal/app/src/main/java/com/creator/podoalglobal/util/ProgressUtil.java
package com.creator.podoalglobal.util;

import android.app.Dialog;
import android.content.Context;
import android.view.Window;

import com.creator.podoalglobal.R;

/**
 * Created by skcrackers on 5/27/16.
 */
public class ProgressUtil extends Dialog {
    public ProgressUtil(Context context) {
        super(context);
        requestWindowFeature(Window.FEATURE_NO_TITLE);
        setContentView(R.layout.view_progress_util);
    }
}

File: 2023/IpjuMadang/app/src/main/java/com/creator/ipjumadang/util/ProgressUtil.kt
package com.creator.ipjumadang.util

import android.app.Dialog
import android.content.Context
import android.view.Window
import com.creator.ipjumadang.R

/**
 * Created by skcrackers on 5/27/16.
 */
class ProgressUtil(context: Context?) : Dialog(context!!) {
    init {
        requestWindowFeature(Window.FEATURE_NO_TITLE)
        setContentView(R.layout.view_progress_util)
    }
}